generate_qt_mappings.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+"""
+This script generated the Qt mapping file according to given Qt include
+directory
+
+Example usage :
+
+   $ ./generate_qt_mappings.py /usr/include/x86_64-linux-gnu/qt5 qt5_11.imp
+
+"""
+
+import argparse
+import glob
+import os
+import re
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("qt_include_dir", help="Qt include directoy")
+    parser.add_argument("output_file", help="Generated output mapping file")
+    parser.add_argument("--warnings", action="store_true", help="Display possible warning messages")
+    args = parser.parse_args()
+
+    headers = glob.glob(os.path.join(args.qt_include_dir, '**/*[!\\.h]'))
```

Comment:
Move this to just before the `for` loop

Reply:
Also, do you think this glob expression works on Windows? Do you have any way to test?

Reply:
The script now works on Windows f15e0dc